Moya-Argo.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Moya-Argo"
-  s.version          = "2.0.0"
+  s.version          = "3.0.0"
```

Comment:
Thanks! Makes sense indeed to bump up only patch version, since the backward compatibility holds.